rules/health-and-resolve.md

```diff
@@ -86,9 +86,9 @@ Twitch is a level 9 Ysoki Mechanic. Her Constitution score is 12, giving a bonus
 
 Grax is a level 4 Vesk Barbarian. His Constitution bonus is +3 (which is also his key ability bonus). Vesks are sturdy and get 6 racial HP. Barbarians have a d12 hit die in D&D, so in Star5er they get 4 HP per level and 3+Con SP per level. So we can calculate:
 
-* Grax has half her levels (4 halved, i.e. 2) plus his key ability modifier (i.e +3). So she has a maximum of 2+3 = 5 RP
-* Grax gets 6 HP from his race, and 4 HP per each of his 4 levels. So she has a Max HP of 6 + 4×4 = 22 HP.
-* For each of his 4 levels, Grax gets 3 SP + 3 bonus SP given his constitution. So she has a Max SP of (3 + 3)×4 = 24 SP.
+* Grax has half his levels (4 halved, i.e. 2) plus his key ability modifier (i.e +3). So he has a maximum of 2+3 = 5 RP
+* Grax gets 6 HP from his race, and 4 HP per each of his 4 levels. So he has a Max HP of 6 + 4×4 = 22 HP.
+* For each of his 4 levels, Grax gets 3 SP + 3 bonus SP given his constitution. So he has a Max SP of (3 + 3)×4 = 24 SP.
```

Comment:
I don't know if Grax is a she or a he, but there are 7 male and 4 female pronouns, so I chose by majority :P